guides/source/active_record_querying.md

```diff
@@ -1266,6 +1266,9 @@ class Client < ActiveRecord::Base
 end
 ```
 
+NOTE: The `default_scope` is also applied while creating/building a record.
+It is not applied while updating a record.
```

Comment:
You are right, I look for the code inside the note and not after  :expressionless:

Updated with example of code.